config.json

```diff
@@ -2,34 +2,54 @@
   "version": "0.0.1",
   "choices": [
     {
-      "type": "database",
-      "architect-file": "https://raw.githubusercontent.com/architect-team/template-configs/main/databases/mysql-architect.yml",
-      "name": "MySQL"
+      "type": "frontend",
+      "repository": "https://github.com/architect-templates/react",
+      "name": "React"
     },
     {
-      "type": "database",
-      "architect-file": "https://raw.githubusercontent.com/architect-team/template-configs/main/databases/mariadb-architect.yml",
-      "name": "MariaDB"
+      "type": "frontend",
+      "repository": "https://github.com/architect-templates/nuxt",
+      "name": "Nuxt"
     },
     {
-      "type": "database",
-      "architect-file": "https://raw.githubusercontent.com/architect-team/template-configs/main/databases/postgres-architect.yml",
-      "name": "Postgres"
+      "type": "backend",
+      "repository": "https://github.com/architect-templates/nestjs",
+      "name": "Nest"
     },
     {
-      "type": "frontend",
-      "repository": "https://github.com/architect-templates/react",
-      "name": "React"
+      "type": "backend",
+      "repository": "https://github.com/architect-templates/django",
+      "name": "Django"
     },
     {
       "type": "backend",
-      "repository": "https://github.com/architect-templates/node-rest-api",
-      "name": "NodeJS"
+      "repository": "https://github.com/architect-templates/flask",
+      "name": "Flask"
     },
     {
       "type": "backend",
       "repository": "https://github.com/architect-templates/go",
       "name": "Go"
+    },
+    {
+      "type": "backend",
+      "repository": "https://github.com/architect-templates/springboot",
+      "name": "Springboot"
+    },
+    {
+      "type": "backend",
+      "repository": "https://github.com/architect-templates/next",
+      "name": "Nextjs"
+    },
+    {
+      "type": "backend",
+      "repository": "https://github.com/architect-templates/next",
+      "name": "Nextjs"
+    },
```

Comment:
Next is here twice and I think it's a frontend but may want to confirm that part 😛

Reply:
ahh, good catch, thanks!